tedana/workflows/tedana.py

```diff
@@ -204,16 +191,38 @@ def _get_parser():
                           default=False)
     optional.add_argument('-v', '--version', action='version', version=verstr)
     parser._action_groups.append(optional)
+
+    rerungrp = parser.add_argument_group('arguments for rerunning the workflow')
+    rerungrp.add_argument('--mix',
+                          dest='mixm',
+                          metavar='FILE',
+                          type=lambda x: is_valid_file(parser, x),
+                          help=('File containing mixing matrix. If not '
+                                'provided, ME-PCA & ME-ICA is done.'),
+                          default=None)
+    rerungrp.add_argument('--ctab',
+                          dest='ctab',
+                          metavar='FILE',
+                          type=lambda x: is_valid_file(parser, x),
+                          help=('File containing a component table from which '
```

Comment:
Not going to disapprove this pull request over it, but I don't think you need the parentheses. Or is that style preferred?

Reply:
I don't know which is more Pythonic, but I just find the parentheses make multiline strings more readable, even when they're not necessary.

Reply:
Okie dokie, I prefer the other way but that's fine, they're both stylistically valid.